mcc/odin/odin/snapshot_list.go

```diff
@@ -13,6 +13,10 @@ func ListSnapshots(
 	result []*rds.DBSnapshot,
 	err error,
 ) {
-	result = make([]*rds.DBSnapshot, 0)
+	input := &rds.DescribeDBSnapshotsInput{}
+	output, err := svc.DescribeDBSnapshots(input)
+	if err == nil {
```

Comment:
Undefined behaviour. You cannot compare against nil. Also breaks the left-aligned happy path.